ci/deploy.yaml

```diff
@@ -0,0 +1,34 @@
+platform: linux
+image_resource:
+  type: docker-image
+  source:
+    repository: theorf/google-cloud-sdk-helm
+params:
+  SERVICE_ACCOUNT_JSON: ((gcp.service_account_json))
+  IMAGE_TAG: latest
+  REGION: europe-west2
+  DOCKER_REGISTRY:
+  RUNNER_URL:
+  PROJECT_ID:
```

Comment:
The same comment I made in the eq-terraform-gcp pull request about ordering of params applies here. I think the order should be determined by:
- the order in which they're used - `PROJECT_ID` and `REGION` are used in the task so should appear before the other params that are used in the script called by the task.
- related params should go together - so `IMAGE_TAG` should appear immediately after `DOCKER_REGISTRY`.